### DISCUSSION TITLE
Actions not running in merge queue when enqueued by github-actions bot

### DISCUSSION THREAD

**Discussion by achilleas-k** [community/community#70310] [Actions] [answered]
### Select Topic Area

Question

### Body

We have a GitHub action set up to automatically approve and enqueue (add to the merge queue) dependabot PRs.  The action is defined [here](https://github.com/osbuild/images/blob/5c15721cb9ea1f98714030fa83112a9fc874f65c/.github/workflows/auto-merge-dependabot.yml) based on [Automating Dependabot with GitHub Actions](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request).  But when the action adds the PR to the merge queue, the actions for the merge queue aren't started and the PR ends up timing out and being removed from the queue.

Here's an example PR of that happening: https://github.com/osbuild/images/pull/210
Note the sequence of events:
1.  github-actions bot approved these changes
2. github-actions bot enabled auto-merge
3.  github-actions bot added this pull request to the merge queue
4. github-merge-queue bot removed this pull request from the merge queue due to no response for status checks

Looking at the actions while the PR is on the merge queue shows that the actions aren't started.

Is this a permission issue?  A problem with the merge strategy or merge queue?

**Comment by achilleas-k** [comment, marked as answer]
Trying a different token because this might be due to the bot token not triggering actions as a protection against recursive triggering: https://github.com/osbuild/images/pull/212

**Reply by achilleas-k** [reply]
This seems to have solved the issue: https://github.com/osbuild/images/pull/213

**Comment by timothyfroehlich** [comment]
Thank you, I think you've solved my problems!
